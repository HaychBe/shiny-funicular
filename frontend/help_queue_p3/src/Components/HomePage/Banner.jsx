
const Banner = () => {
    return(
        <h1>NAV OR BANNER HERE</h1>
    );
}

export default Banner;