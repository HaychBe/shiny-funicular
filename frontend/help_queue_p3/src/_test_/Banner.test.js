import '@testing-library/jest-dom';
import {create} from 'react-test-renderer';
import Banner from '../Components/HomePage/Banner';


  describe(`Test of Banner`, () => {

    test(`Banner should match the snapshot every time it is rendered`, () => {
      const instance = create(<Banner/>).toJSON(); 
      expect(instance).toMatchSnapshot();
    });
  });