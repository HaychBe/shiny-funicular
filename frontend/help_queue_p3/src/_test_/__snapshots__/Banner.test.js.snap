// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Banner Banner should match the snapshot every time it is rendered 1`] = `
<div>
  <img
    alt="Logo"
    id="mainBanner"
    src="Banner.png"
  />
  ;
</div>
`;
